websites/D/Durarara/dist/metadata.json

```diff
@@ -0,0 +1,20 @@
+{
+  "author": {
+    "name": "Dawkin Nguyen",
+    "id": "414274321779916800"
+  },
+  "service": "Durarara!!",
```

Comment:
service name must be the same as folder name

Reply:
ok